app/services/cocina_generator/description/generator.rb

```diff
@@ -194,7 +127,10 @@ def admin_metadata
           ]
         )
       end
-      # rubocop:enable Metrics/MethodLength
+
+      def admin_metadata_creation_date
+        work_version.work.work_versions.first&.published_at || work_version.work.created_at
```

Comment:
When you call `work_versions` is the order guaranteed?  I'm not sure it is.

Reply:
Wouldn't it be in primary key order by default?

Reply:
I think that usually ends up being the case, but iirc, there's no guaranteed ordering by default for SQL queries if no order is explicitly specified.  So I think it's possible that the result wouldn't be ordered by primary key (my very gut feeling is that 90-ish% of the queries I've run against a PG/oracle/mysql have come back in PK order if no sort was specified -- but I also think I've seen occasional query results that break that expectation).

ha, of course i should've just gone to the PG manual instead of dredging my memory:
https://www.postgresql.org/docs/12/queries-order.html 💬 
> After a query has produced an output table (after the select list has been processed) it can optionally be sorted. If sorting is not chosen, the rows will be returned in an unspecified order. The actual order in that case will depend on the scan and join plan types and the order on disk, but it must not be relied on. A particular output ordering can only be guaranteed if the sort step is explicitly chosen.

Reply:
Ordering now done https://github.com/sul-dlss/happy-heron/pull/2417/files#diff-ebe838f9d71d26b6227d6d37c96a7e2d93fb68a80d55075c7f2374bfcb4688abR14

It seems sensible to have this as default ordering for this association.